aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -419,7 +566,10 @@ Tensor sparse_compressed_to_dense(
     dense = dense.reshape({n_batch, -1, values.size(-2), values.size(-1)});
     for (auto batch : c10::irange(n_batch)) {
       Tensor batch_indices = at::_convert_indices_from_csr_to_coo(
-          crow_indices[batch], col_indices[batch], false, false);
+          compressed_indices[batch],
+          plain_indices[batch],
+          false,
+          self.layout() == kSparseBsc);
```

Comment:
A bit cryptic. What does `Bsr` have to do with the method `from_csr_to_coo`?

Reply:
`_convert_indices_from_csr_to_coo` is poorly named. The method converts the compressed indices to coo indices. The coo indices will be correct for whatever axis is compressed. 

It should probably be re-named.